test/parsing_spec.rb

```diff
@@ -1,7 +1,178 @@
-require_relative './helper.rb'
-require 'time'
-require 'test/unit'
+require_relative 'spec_helper'
 
+def set_now_to now
+  Timecop.freeze now
+end
+
+describe "parsing strings to get timeframes" do
+
+  let(:parse) { ->(x) { Tickle.parse x } }
+
+  let(:date_matcher) { ->(x, y) { x.to_i.must_equal y.to_i } }
+
+  describe "the basics" do
+
+    [
+      Time.parse('2015-04-27 15:19:14 -0500'),
+      Time.parse('2015-01-01 15:19:14 -0500'),
+      Time.parse('2017-12-25'),
```

Comment:
I'm purposely starting my tests from multiple date points, as some bugs would only appear if the tests started from a day > 15, or a month > 9.